pages/your_tasks/organising_software_projects.md

```diff
@@ -82,19 +83,25 @@ project_name/
 │   ├── main_script.py    # main script/code entry point
 │   └── ...
 ├── doc/                  # documentation for your software
-├── index.html            # entry point into the documentation website    
+│   ├── index.html            # entry point into the documentation website    
+│   └── ...
 └── ...
 ```
 
-Checkout the {% tool 'fair-python-coockiecutter' %} - a template tool that can help you set up a Python software project skeleton that uses modern state-of-the-art development tools and helps you follow best practices for code and metadata quality.
+For Python:
+
+- Checkout the {% tool 'fair-python-coockiecutter' %} - a template tool that can help you set up a Python software project skeleton that uses modern state-of-the-art development tools and helps you follow best practices for code and metadata quality.
+- {% tool "poetry" %} is a popular tool for managing dependencies in Python projects which you can also use to create a new directory with the necessary structure if you are starting a new Python project from scratch.
```

Comment:
These templates do a lot more than just creating a directory structure. Neither of them make a complete research-project structure as the one sketched above, they only do the software package parts (src, doc, tests; but not e.g. data, results, manuscript). I feel like these links would fit better in the https://everse.software/RSQKit/languages_tools_infrastructures task page. At the bottom, there's a section on templates, we could add this paragraph there. If at some point we want to make a separate "Use templates" task page, we can simply take all that out in one go.

Reply:
I agree with both points - I'll try to reword the text to make it clearer these tools do other things and they help with project structure "on the side". And I agree we should add them to the https://everse.software/RSQKit/languages_tools_infrastructures task page (it is OK to repeat and mention them elsewhere too) - do you want to take this second task @egpbos?

Reply:
OK, I have now addresses the comment about better explaining the role of poetry and cookiecutter in organsing directory structure via 32310a42d36ee6089c19ef8b7e5ab8c0a58b53b6. @egpbos - if you'd like to review and see if you can approve and then you can address the other issue separately in the https://everse.software/RSQKit/languages_tools_infrastructures task page.

Reply:
Indeed, there was already an issue, #91.